stl/inc/complex

```diff
@@ -78,7 +79,7 @@ namespace _Float_multi_prec {
 
     // 1x precision + 1x precision -> 2x precision
     // the result is exact when:
-    // 1) the result doesn't overflow
+    // 1) no internal overflow occurs

```

Comment:
Could you explain this change? I don't understand how these are flipped (or what they mean...)

Reply:
For `_Add_small_x2(Float, Float)` (aka `Fast2Sum`), if `_Xval + _Yval` doesn't cause floating point overflow, `_Add_small_x2` won't suffer from any overflow.

For `_Add_x2(Float, Float)` (aka `2Sum`) however, there are cases when `_Xval + _Yval` doesn't overflow, but `_Add_x2`'s other internal operation overflows. For example, with `_Xval = -0x1.8p+971` and `_Yval = numeric_limits<double>::max()`, `_Add_x2` returns a NaN because of overflow when computing `_Ymod`.